estate/models/estate_property.py

```diff
@@ -0,0 +1,25 @@
+from odoo import fields, models
+from datetime import date , timedelta
+
+class EstateProperty(models.Model):
+    _name = "estate.property"
+    _description = "Real Estate App"
+
+    name = fields.Char(required=True)
+    description = fields.Text() 
+    postcode = fields.Char()
+    date_availability = fields.Date(default=lambda self: date.today() + timedelta(days=90))
+    expected_price = fields.Float()
+    selling_price = fields.Float(readonly=True , copy=False)
```

Comment:
```suggestion
    selling_price = fields.Float(copy=False)
```
If i make it readonly how will i be able to assign value to it as there is no compute method.